docs/deploy/vitis_ai.rst

```diff
@@ -541,20 +543,50 @@ TVM.
    import tvm
    import tvm.relay as relay
    from tvm.contrib.target import vitis_ai
-   from tvm.contrib import util, graph_runtime
+   from tvm.contrib import utils, graph_runtime
    from tvm.relay.build_module import bind_params_by_name
    from tvm.relay.op.contrib.vitis_ai import annotation
 
 After importing a convolutional neural network model using the usual
 Relay API's, annotate the Relay expression for the given Vitis-AI DPU
 target and partition the graph.
 
+.. note::
+
+    We recommend switching DPU convolutions' data layouts to NHWC and CPU comvolutions'
```

Comment:
```suggestion
    We recommend converting DPU convolutions' data layouts to NHWC and CPU convolutions'
```

In fact, to get the best performance by getting rid of the layout transform overheads, we should suggest using NHWC for an entire model. For the Conv2D that remains on CPU, we could use auto_scheduler to tune its performance, and it could be even better than the tuned Conv2D with NCHW data layout. It might be better to mention this in the doc, and point to the auto_scheduler tutorials.